test/org/sagebionetworks/bridge/services/AppConfigServiceTest.java

```diff
@@ -255,13 +391,29 @@ public void getAppConfigForUserReturnsOldestVersion() {
     @Test
     public void createAppConfig() {
         when(mockStudyService.getStudy(TEST_STUDY)).thenReturn(study);
+        when(mockSchemaService.getUploadSchemaByIdAndRev(any(), any(), anyInt())).thenReturn(mockUploadSchema);
+        when(mockSurveyService.getSurvey(any(), any(), anyBoolean(), anyBoolean())).thenReturn(mockSurvey);
+        when(mockAppConfigElementService.getElementRevision(any(), any(), anyLong())).thenReturn(mockConfigElement);
+        
+        when(mockSurvey.isPublished()).thenReturn(true);
         
         AppConfig newConfig = setupAppConfig();
+        newConfig.setClientData(TestUtils.getClientData());
+        newConfig.setSurveyReferences(SURVEY_REF_LIST);
+        newConfig.setSchemaReferences(SCHEMA_REF_LIST);
+        newConfig.setConfigReferences(CONFIG_REF_LIST);
         
         AppConfig returnValue = service.createAppConfig(TEST_STUDY, newConfig);
+        
         assertEquals(TIMESTAMP.getMillis(), returnValue.getCreatedOn());
         assertEquals(TIMESTAMP.getMillis(), returnValue.getModifiedOn());
         assertEquals(GUID, returnValue.getGuid());
+        assertEquals(newConfig.getLabel(), returnValue.getLabel()); //
+        assertEquals(TEST_STUDY.getIdentifier(), returnValue.getStudyId()); //
+        assertEquals(TestUtils.getClientData().toString(), returnValue.getClientData().toString());
```

Comment:
Minor: If these are the same object, then the .toString() will also be the same. That said, it might just be cleaner to test the objects themselves are the same instead of doing string comparison.